components/ui/Heading.tsx

```diff
@@ -0,0 +1,329 @@
+import { PropsWithChildren } from "react";
+
+type fontSize =
```

Comment:
just a remark that capitalized names for types are used through the app. this is internal type so it doesn't matter in this case.